test/Microsoft.AspNetCore.Routing.Tests/Matchers/DispatcherAssert.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+using Xunit.Sdk;
+
+namespace Microsoft.AspNetCore.Routing.Matchers
+{
+    internal static class DispatcherAssert
```

Comment:
I tend to build this sort of thing for every significant project I start nowadays. IMO it pays for itself many times.

Reply:
Ok! Interested to see how it gets used.